### ISSUE TITLE
Multi label classification 

### ISSUE THREAD

**Issue by adavoudi** [sseung0703/Knowledge_distillation_methods_wtih_Tensorflow#9]
Hi, 

Thanks for your amazing work. I have a question regarding Multi-label classification. For this purpose, should we just replace any softmax with sigmoid in the loss functions?

for example in the `DML` function, should I change it from:

```python
def DML(student, teacher):
    '''
    Ying Zhang, Tao Xiang, Timothy M. Hospedales, Huchuan Lu. 
    Deep mutual learning.
    IEEE Conference on Computer Vision and Pattern Recognition. 2018.
    '''
    with tf.variable_scope('KD'):
        return (tf.reduce_mean(tf.reduce_sum(tf.nn.softmax(teacher)*(tf.nn.log_softmax(teacher)-tf.nn.log_softmax(student)),1)) +
                tf.reduce_mean(tf.reduce_sum(tf.nn.softmax(student)*(tf.nn.log_softmax(student)-tf.nn.log_softmax(teacher)),1)))/2
```

to this:

```python
def DML(student, teacher):
    '''
    Ying Zhang, Tao Xiang, Timothy M. Hospedales, Huchuan Lu. 
    Deep mutual learning.
    IEEE Conference on Computer Vision and Pattern Recognition. 2018.
    '''
    with tf.variable_scope('KD'):
        return (tf.reduce_mean(tf.reduce_sum(tf.sigmoid(teacher)*(tf.log_sigmoid(teacher)-tf.log_sigmoid(student)),1)) +
                tf.reduce_mean(tf.reduce_sum(tf.sigmoid(student)*(tf.log_sigmoid(student)-tf.log_sigmoid(teacher)),1)))/2
```
Is that correct?

Thank you in advance

**Comment by sseung0703** [comment]
Thank you for your attention to my work. :)


I think your modification is OK, but I don't know how much it is effective.

So, you should do experiments yourself to confirm.
